syncd/syncd.cpp

```diff
@@ -2857,6 +2873,29 @@ void processFlexCounterEvent(
 
                 FlexCounter::setQueueAttrList(vid, rid, groupName, queueAttrIds);
             }
+            else if (objectType == SAI_OBJECT_TYPE_INGRESS_PRIORITY_GROUP && field == PG_COUNTER_ID_LIST)
+            {
+                std::vector<sai_ingress_priority_group_stat_t> pgCounterIds;
+                for (const auto &str : idStrings)
+                {
+                    sai_ingress_priority_group_stat_t stat;
+                    sai_deserialize_ingress_priority_group_stat(str, stat);
+                    pgCounterIds.push_back(stat);
+                }
+                FlexCounter::setPriorityGroupCounterList(vid, rid, groupName, pgCounterIds);
+            }
+            else if (objectType == SAI_OBJECT_TYPE_INGRESS_PRIORITY_GROUP && field == PG_ATTR_ID_LIST)
```

Comment:
Is PG_ATTR_ID_LIST field used by watermark?

Reply:
It's not used by watermark, but the other objects types (queue, port) have both  SAI_STAT and SAI_ATTR handlers, so implemented it for priority-groups as well.